po/pt.po

```diff
@@ -1,131 +1,160 @@
-# Portuguese translations for com.github.cassidyjames.ideogram package.
-# Copyright (C) 2019 THE com.github.cassidyjames.ideogram'S COPYRIGHT HOLDER
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
 # This file is distributed under the same license as the com.github.cassidyjames.ideogram package.
-# Automatically generated, 2019.
+# André Barata <andretiagob@protonmail.com>, 2020.
 #
 msgid ""
 msgstr ""
 "Project-Id-Version: com.github.cassidyjames.ideogram\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2019-11-06 20:45+0000\n"
-"PO-Revision-Date: 2019-01-25 19:01-0700\n"
-"Last-Translator: Automatically generated\n"
+"POT-Creation-Date: 2020-02-25 20:16+0000\n"
+"PO-Revision-Date: 2020-08-26 15:01+0100\n"
 "Language-Team: none\n"
-"Language: pt\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.4.1\n"
+"Last-Translator: André Barata <andretiagob@protonmail.com>\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: pt\n"
 
 #: data/com.github.cassidyjames.ideogram.appdata.xml.in:7
 #: data/launcher.desktop.in:3 src/MainWindow.vala:33
 msgid "Ideogram"
-msgstr ""
+msgstr "Ideogram"
 
 #: data/com.github.cassidyjames.ideogram.appdata.xml.in:8
 msgid "Insert emoji anywhere 🎉"
-msgstr ""
+msgstr "Insira símbolos de emojis em todo o lado 🎉"
 
 #: data/com.github.cassidyjames.ideogram.appdata.xml.in:10
 msgid ""
 "Quickly insert emoji anywhere you can paste text, including non-native apps. "
 "Hit ⌘+E to open the emoji picker, choose the emoji you want, and it’s "
 "instantly copied and pasted into any focused text input. 👍"
 msgstr ""
+"Insira emojis rapidamente em qualquer lado que conseguir colar texto, "
+"aplicações não nativas incluídas. Pressione ⌘+E para abrir o selecionador de "
+"emojis, escolha o emoji que quiser e este é copiado e colado "
+"instantaneamente para dentro de uma entrada de texto. 👍"
 
 #: data/com.github.cassidyjames.ideogram.appdata.xml.in:11
 msgid ""
 "Not feeling the default keyboard shortcut? Head to System Settings → "
-"Keyboard → Shortcuts → Custom to set it to anything you’d like."
+"Keyboard → Shortcuts → Custom to set it to anything you’d like. 😀️"
 msgstr ""
+"Não gosta dos atalhos padrão do teclado? Vá a Definições de Sistema → "
+"Teclado → Atalhos → Personalizar para definir o que quiser ao seu gosto. 😀️"
 
 #: data/com.github.cassidyjames.ideogram.appdata.xml.in:20
+msgid "Refreshed icons thanks to Micah Ilbery"
+msgstr "Ícones melhorados graças a Micah Ilbery"
```

Comment:
I suspect "melhorados" ("improved" in English) indeed matches the intent of the original text (@cassidyjames, can you confirm?) but in any case it would be better to stick to what the original _actually_ says; so I'd go with something like this:

```suggestion
msgstr "Ícones atualizados por Micah Ilbery"
```

Reply:
Done. Changed the other "thanks to" too as well